src/jsonrpc/miner_jsonrpc_sc.erl

```diff
@@ -9,15 +9,15 @@
 handle_rpc(<<"sc_active">>, []) ->
     case (catch blockchain_state_channels_server:active_sc_ids()) of
         {'EXIT', _} -> ?jsonrpc_error(timeout);
-        undefined -> #{<<"active">> => <<"none">>};
+        undefined -> #{<<"active">> => null};
         BinIds -> #{<<"active">> => [ ?TO_VALUE(base64:encode(I)) || I <- BinIds ]}
     end;
 handle_rpc(<<"sc_active">>, Params) ->
     ?jsonrpc_error({invalid_params, Params});
 handle_rpc(<<"sc_list">>, []) ->
     case (catch blockchain_state_channels_server:state_channels()) of
         {'EXIT', _} -> ?jsonrpc_error(timeout);
-        undefined -> #{<<"channels">> => <<"none">>};
+        undefined -> #{<<"channels">> => null};
```

Comment:
Use `null` where appropriate.

Reply:
Would it be more appropriate to return an empty list here? (and anywhere else where a list is returned)?

Reply:
Yes!

Reply:
Fixed in 0c0e68b